docs/user-guide/jobs.md

```diff
@@ -323,6 +308,70 @@ Here, `W` is the number of work items.
 | Single Job with Static Work Assignment                                     |          W          |        any           |
 
 
+## Advanced Usage
+
+### Specifying your own pod selector
+
+Normally, when you create a job object, you do not specify `spec.selector`.
+The system defaulting logic adds this field when the job is created.
+It picks a selector value that will not overlap with any other jobs.
+
+However, in some cases, you might need to override this automatically set selector.
+To do this, you can specify the `spec.selector` of the job.
+
+Be very careful when doing this.  If you specify a label selector which is not
+unique to the pods of that job, and which matches unrelated pods, then pods of the unrelated
+job may be deleted, or this job may count other pods as completing it, or one or both
+of the jobs may refuse to create pods or run to completion.  If a non-unique selector is
+chosen, then other controllers (e.g. ReplicationController) and their pods may behave
+in unpredicatable ways too.  Kubernetes will not stop you from making a mistake when
+specifying `spec.selector`.
+
+Here is an example of a case when you might want to use this feature.
+
+Say job `old` is already running.  You want existing pods
+to keep running, but you want the rest of the pods it creates
+to use a different pod template and for the job to have a new name.
+You cannot update the job because these fields are not updatable.
+Therefore, you delete job `old` but leave its pods
+running, using `kubectl delete jobs/old-one --cascase=false`.
+Before deleting it, you make a note of what selector it uses:
```

Comment:
Shouldn't you first create the new job? Otherwise, a completion could happen while no job is running?